birdhouse/components/stac/config/magpie/config.yml.template

```diff
@@ -0,0 +1,20 @@
+providers:
+  # definition of STAC service for API access
+  stac:
+    url: http://stac:8000
+    title: STAC
+    public: true
+    c4i: false
+    type: api
+    sync_type: api
+
+permissions:
+  # Administrator permissions over whole service
+  - service: stac
+    permission: read
+    group: anonymous
```

Comment:
the comment above says "Administrator permissions over whole service" but this is giving anonymous permissions. It seems like a bad idea to give anonymous write permissions here.